src/test/java/com/google/firebase/remoteconfig/FirebaseRemoteConfigClientImplTest.java

```diff
@@ -63,7 +63,33 @@
           404, ErrorCode.NOT_FOUND,
           500, ErrorCode.INTERNAL);
 
-  private static final String MOCK_TEMPLATE_RESPONSE = "{\"conditions\": [], \"parameters\": {}}";
+  private static final String MOCK_TEMPLATE_RESPONSE = "{"
```

Comment:
Probably best to load this from a resource file.

Reply:
Good call!